plugin/kubernetes/kubernetes.go

```diff
@@ -263,6 +263,15 @@ func (k *Kubernetes) InitKubeCache() (err error) {
 		k.opts.selector = selector
 	}
 
+	if k.opts.namespaceLabelSelector != nil {
```

Comment:
I think the namespace option and the namespaceLabelSelector should be mutually exclusive. If both are set we should either error or only use one and document it

Reply:
I saw you did that later in option parsing, 👍